src/l500/l500-color.cpp

```diff
@@ -141,6 +141,13 @@ namespace librealsense
         color_ep->register_metadata(RS2_FRAME_METADATA_LOW_LIGHT_COMPENSATION, make_attribute_parser(&md_rgb_control::low_light_comp, md_rgb_control_attributes::low_light_comp_attribute, md_prop_offset));
         color_ep->register_metadata(RS2_FRAME_METADATA_FRAME_TIMESTAMP, make_uvc_header_parser(&platform::uvc_header::timestamp));
 
+        // We manipulate several controls when the CAH process turn on/off the color stream
+        // This should be done after registration of the device options
+        if (_autocal)
```

Comment:
We have a whole block above that is if _autocal is non-null -- can me move this up?

Reply:
Oh it's because it needs to be done after option registration?

Reply:
> Oh it's because it needs to be done after option registration?

Yes, I wanted to make sure it will be done after all registrations, maybe more controls will be added in the future.

I also added a comment that explain why I did it.